app/src/main/java/com/example/perfectweatherallyear/utils/ForecastService.kt

```diff
@@ -38,19 +39,17 @@ class ForecastService : Service() {
     }
 
     override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
-        var notification = Notification()
-        mRunnable = Runnable{
-            jobNotification = CoroutineScope(Dispatchers.Default).launch {
-                notification = NotificationHandler.createNotification(
-                    application,
-                    applicationContext,
-                    loadHourWeatherTempForNotification()
-                )
-                startForeground(NOTIFICATION_ID, notification)
-                delay(5000)
+        notification = NotificationHandler.createNotification(application, applicationContext, getString(R.string.notification_text))
+        startForeground(START_NOTIFICATION_SERVICE_ID, notification)
+
+        CoroutineScope(Dispatchers.Default).launch {
+            notification = when (val notificationTextResult = loadHourWeatherTempForNotification()) {
```

Comment:
Again the same error :) Please do not concentrate on the notification object. Remember the algorithm that we've discussed.